packages/expo-apple-authentication/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### 💡 Others
 
+- Extend the `AppleAuthenticationButton` component by `View`
```

Comment:
- Extend the `AppleAuthenticationButton` component type by the `View` component type. ([#13567](https://github.com/expo/expo/pull/13567) by [@Simek](https://github.com/Simek))